.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -101,8 +101,8 @@ body:
       For TypeScript:
       ```sh
       node --version
-      npm list @coinbase/agentkit
-      npm list @coinbase/agentkit-langchain  # if using LangChain extension
+      pnpm -r list @coinbase/agentkit
```

Comment:
`npm list` is recursive by default, while `pnpm list` is not. Hence, the added `-r` flag to recurse